nomad/status_endpoint.go

```diff
@@ -78,6 +78,31 @@ func (s *Status) Peers(args *structs.GenericRequest, reply *[]string) error {
 	return nil
 }
 
+// RPCServers is used to get all the RPC server addresses in a region
+func (s *Status) RPCServers(args *structs.GenericRequest, reply *[]string) error {
```

Comment:
Note to reviewers, I've intentionally left off adding a HTTP API for this endpoint. We already have a `/v1/agent/servers` endpoint but it's an agent endpoint and currently seems to work in a weird and possibly buggy way (`nomad operator api /v1/agent/servers` always returns "wrong method" for me!). It's not necessary to do this to fix the regression, so I'd like to do any polish we might want in a follow-up PR.